renovate.json

```
@@ -1,4 +1,15 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["github>renovatebot/presets-internal:docker-images"]
+  "extends": ["github>renovatebot/presets-internal:docker-images"],
+  "ignorePaths": [],
+  "packageRules": [{
+    "description": "Don't hold back any PRs for approval",
+    "packagePatterns": ["*"],
+    "dependencyDashboardApproval": false
+  }, {
+    "description": "Don't automerge test minor/major because we might want to test old versions too",
+    "paths": ["test/"],
```

```suggestion
    "updateTypes": ["major", "minor"],
```
restrict update types?

---

ok, will need 2 rules